src/components/map/MapService.js

```diff
@@ -424,6 +424,24 @@ goog.require('ga_urlutils_service');
           }
           return olLayer;
         };
+
+        // Make a GetLegendGraphic request
+        this.getLegend = function(layer) {
+          var defer = $q.defer();
+          var params = layer.getSource().getParams();
+          var html = '<img src="' + gaUrlUtils.append(layer.url,
+              gaUrlUtils.toKeyValue({
+            request: 'GetLegendGraphic',
+            layer: params.LAYERS,
+            style: params.style || 'default',
+            service: 'WMS',
+            version: params.version || '1.3.0',
+            format: 'image/png',
```

Comment:
Is the format alwasy fix? Or could it be different and would need to be retrieved from getcap?